bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/internal/AudioServlet.java

```diff
@@ -127,32 +140,50 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
             }
 
             ServletOutputStream os = resp.getOutputStream();
-            IOUtils.copy(stream, os);
+
+            InputStream is = null;
+            if (multiAccess) {
+                // we need to care about concurrent access and have a separate stream for each thread
+                try {
+                    is = ((FixedLengthAudioStream) stream).getClonedStream();
+                } catch (AudioException e) {
+                    resp.sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, e.getMessage());
+                }
+            } else {
+                is = stream;
+            }
+            IOUtils.copy(is, os);
             resp.flushBuffer();
             IOUtils.closeQuietly(stream);
```

Comment:
We should close `is` now to ensure the cloned stream if `multiAccess` is true (if it is false `is` is equal to `stream`).